lib/interface/transactional.js

```diff
@@ -0,0 +1,98 @@
+module.exports=function Transactional(createsend) {
+
+	function txResponseCallback(error, results, callback) {
+		if (error) {
+			callback(error);
+		} else {
+			callback(null, results);
+		}
+	}
+
+	function deleteNullProperties(details) {
+		for (key in details) {
+			if (!details[key]) {
+				delete details[key];
+			}
+		}
+	}
```

Comment:
This could probably just become:

    function deleteNullProperties(details) {
        return details.filter(function(v) { return v; });
    }